contracts/ocr2/src/msg.rs

```diff
@@ -29,25 +29,25 @@ pub struct InstantiateMsg {
 #[serde(rename_all = "snake_case")]
 pub enum ExecuteMsg {
     SetConfig {
-        signers: Vec<Vec<u8>>,
+        signers: Vec<Binary>,
         transmitters: Vec<String>,
         f: u8,
-        onchain_config: Vec<u8>,
+        onchain_config: Binary,
         offchain_config_version: u64,
-        offchain_config: Vec<u8>,
+        offchain_config: Binary,
     },
```

Comment:
@RodrigoAD all of these are now base64 encoded to save space. Depending on the library you're using it might need some changes

Reply:
Let's reuse the same PR to add any Gauntlet changes.